src/resolveGitDir.js

```diff
@@ -0,0 +1,5 @@
+const path = require('path')
+
+module.exports = function resolveGitDir(gitDir) {
+  return gitDir ? path.resolve(gitDir) : process.cwd()
```

Comment:
Is this right? Wouldn't `path.resolve` resolve from current directory(`lint-staged/src`)? Is that what we want? I am thinking we should use `app-root-path` to resolve relative to the `package.json`.

Reply:
> I am thinking we should use app-root-path to resolve relative to the package.json.

Are you talking about the `process.cwd()` case? If so, then probably you're right but I'm not sure and we don't have any issues with this code since it's been inside so I'd avoid changing it for now.

Reply:
Hmm.. probably `path.resolve` uses the directory where `lint-staged` was invoked which would be the location of `package.json`. And yea, no need to change what already works.